index.html

```diff
@@ -76,6 +76,26 @@
     <li>Kopi Po Siu Dai</li>
   </ul>
 </div>
+<div class="wide-container" id="make-own-kopi">
+    <p>Make your own Kopi</p>
+    <p>(the sum of all ingredients must equal 1)</p>
+    <form id="recipe">
+        <label style="padding-right:55px;">water</label>
+        <input type="range" name="water" max="1" min="0" step="0.1" value="0.4" required><span for="water">0.4</span><br/>
```

Comment:
I don't think `for` attribute is valid for `span` tag here 😁 It works, but well, not valid HTML.